.prettierrc

```diff
@@ -6,10 +6,10 @@
     {
       "files": "*.md",
       "options": {
-        "printWidth": 70,
+        "printWidth": 80,
```

Comment:
This value is going to be used for example formatting inside Markdown and may cause line wrapping.

Reply:
Yes, that's why I changed it. I think 80 is a standard length that fits on most modern screens but I can revert as well if there is a good reason to have 70.

Reply:
I mean examples in the Styleguidist, in the source code.